site/content/en/latest/troubleshooting/envoy-proxy-admin-interface.md

```diff
@@ -0,0 +1,42 @@
++++
+title = "Advanced: Envoy Proxy Admin Interface"
++++
+
+## Overview
+Platform admins looking to troubleshoot low level aspects of the data plane such as xDS config and heap dump, can directly connect to the Envoy Proxy Admin Interface.
+
+**Note**: Application Developers may not have access to the namespace where the Envoy Proxy fleet is running and should rely on [exported telemetry](https://gateway.envoyproxy.io/docs/tasks/observability/) instead for troubleshooting.
+
+## Prerequisites
+
+{{< boilerplate prerequisites >}}
+
+### Access
+
+You will need to port-forward to the admin interface port (currently 19000) on the Envoy deployment that corresponds to a Gateway, since it only listens on the `localhost`
```

Comment:
It might be worth to mention that this port needs to be exposed in the deployment and service of EG, as it is not exposed by default
wdyt?

Reply:
I don't think the port needs to be defined in the deployment,  if you check out the steps, it works w/o doing that

Reply:
sorry, I thought this is about the EG deployment, my previous comment is invalid then
This is working fine :)